### ISSUE TITLE
When trying to create a Task Bundle using a TFLite file, I'm not allowed to enter the stop token of the model

### ISSUE THREAD

**Issue by Arya-Hari** [google-ai-edge/mediapipe#5715]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

No

### OS Platform and Distribution

Linux Ubuntu 16.04

### Programming Language and version

Python

### Solution

LLM Inference

### Describe the actual behavior

I created .tflite file using ai-edge-torch for Llama 3.2 1B model and now am trying to deploy it for inference on edge. When trying to create the task bundle, the stop token is asked. When I provide "<|end_of_text|>", it is not able to resolve it. I previously converted the tokenizer to the SentencePiece format through the code given in the ai-edge-torch repository.

### Describe the expected behaviour

The task bundle should be created without errors.

### Standalone code/steps you may have used to try to get what you need

I tried to manually check the possible tokens the model could identify using its vocab and "<|end_of_text|>" is a token in its vocab. 

I also tried changing the stop token and the task bundle was created. However, on using the bundle for deployment, I was getting a Failed to initialize engine : modelError building tflite model. Also, just as a side question, the .task file that's created, can it be used interchangeably with the .bin file that's given in the model path in the repository examples?

**Comment by kuaashish** [comment]
Hi @Arya-Hari,

Could you please share the complete example you are using from our documentation? Additionally, if you have any error logs, sharing them would help us better understand the issue.

Thank you!!

**Comment by Arya-Hari** [comment]
@kuaashish Hello...is there way to resolve this?

**Comment by talumbau** [comment]
Thanks for all of the detail provided. Two quick items:

1. re: `.task` vs. `.bin`: Yes, you can use the `.task` wherever you would use a `.bin` file. The `.task` extension indicates that the file is a "converted TF Lite model + metadata/tokenizer"
2. I noticed in your provided script that you have this line:

```
tflite_model="/content/gemma_2b_quantized.tflite" # @param {type:"string"}
```

Is this just a copy/paste error? I assumed you would have something like `llama3_1_1b_quantized.tflite`, not a Gemma model.

**Comment by Arya-Hari** [comment]
Hi @talumbau. To clarify, I used the quantization script provided in the [AI Edge Torch repository](https://github.com/google-ai-edge/ai-edge-torch/tree/main/ai_edge_torch/generative/quantize) for quantizing and converting it to the TFLite format. The script used there, by default, saves the output file under the name of ```gemma_2b_quantized.tflite```. I forgot to change it before using it. But, I changed everything else from the script to work for Llama 3.2 1B instead. Sorry for the confusion.
